modules/47_regipol/regiCarbonPrice/preloop.gms

```diff
@@ -93,6 +93,21 @@ $ifthen.cm_implicitPePriceTarget not "%cm_implicitPePriceTarget%" == "off"
   p47_implicitPePriceTax(t,regi,entyPe) = 0;
   
 $endIf.cm_implicitPePriceTarget
+$ENDIF.ESprice
+$offtext
+
+*** Increase SE2FE efficiency for gases in DEU following AGEB data from 2020
```

Comment:
please create an issue to move this to an input file in the near future.

Reply:
Added to issue https://github.com/remindmodel/remind/issues/1032